app/src/test/groovy/gov/hhs/cdc/trustedintermediary/etor/demographics/PatientDemographicsControllerTest.groovy

```diff
@@ -53,84 +54,49 @@ class PatientDemographicsControllerTest extends Specification {
         then:
         response.getBody() == mockBody
         response.getStatusCode() == 200
-        response.getHeaders().get(PatientDemographicsController.CONTENT_TYPE_LITERAL) == PatientDemographicsController.APPLICATION_JSON_LITERAL
-    }
-
-    def "demographics constructResponse fails to make the JSON"() {
-
-        given:
-        def formatter = Mock(Jackson)
-        formatter.convertToJsonString(_ as PatientDemographicsResponse) >> { throw new FormatterProcessingException("couldn't make the JSON", new Exception()) }
-        TestApplicationContext.register(Formatter, formatter)
-
-        TestApplicationContext.injectRegisteredImplementations()
-
-        when:
-        def response = PatientDemographicsController.getInstance().constructResponse(new PatientDemographicsResponse("asdf-12341-jkl-7890", "asdf1234"))
-
-        then:
-        response.statusCode == 500
     }
 
-    def "failed constructResponse works"() {
-
+    def "demographics constructResponse with error string works"() {
         given:
+        def mockStatusCode = 500
         def mockBody = "DogCow goes Moof"
-        def mockResponseStatus = 404
+        def response = new DomainResponse(mockStatusCode)
+        response.setBody(mockBody)
 
-        def formatter = Mock(Jackson)
-        formatter.convertToJsonString(_ as Map) >> mockBody
-        TestApplicationContext.register(Formatter, formatter)
+        def domainResponseHelper = Mock(DomainResponseHelper)
+        domainResponseHelper.constructErrorResponse( mockStatusCode, _ as String) >> { response }
```

Comment:
You actually don't need to make this a closure (lambda) with the `{` and `}`.  You just `>>` the value directly.  You only need a closure (lambda) when you need to execute code.

```suggestion
        domainResponseHelper.constructErrorResponse( mockStatusCode, _ as String) >> response
```